"Error: Cannot find module 'kss/builder' "

Starting with v 3.0.0-beta.19, getting the error 

`Error: Cannot find module 'kss/builder'`

When attempting to load kss.

It seems to be looking in a different place - or in a different way - as of [this commit](https://github.com/kss-node/kss-node/commit/4c6502f525e9b93d385c6acab91964e83f074c2a)

--- a/lib/cli.js
+++ b/lib/cli.js
@@ -130,21 +130,9 @@ const cli = function(opts) {
 
   // Confirm this is a compatible builder.
   return checkBuilderPath.then(() => {
-    return KssBuilderBase.loadBuilder(options.builder);
+    return KssBuilderBase.loadBuilder(KssBuilderBase.builderResolve(options.builder));
   }).catch(error => {
-    let catchLoadFailure;
-    // If the builder specified was "builder/twig" and there was no existing
-    // path relative to the working directory, we should try again while using
-    // the path to kss-node's Twig builder.
-    // istanbul ignore if
-    if (supportedBuilders.indexOf(options.builder) > -1) {
-      options.builder = path.resolve(__dirname, '..', options.builder);
-      catchLoadFailure = KssBuilderBase.loadBuilder(options.builder);
-    } else {
-      catchLoadFailure = Promise.reject(error);
-    }
-
-    return catchLoadFailure.catch(error => {
+    return Promise.reject(error).catch(error => {
       reportError(error);
       throw error;
     });

--- a/lib/kss.js
+++ b/lib/kss.js
@@ -63,7 +63,7 @@ const kss = function(options) {
   let builder;
 
   // Confirm this is a compatible builder.
-  return KssBuilderBase.loadBuilder(options.builder || require('../builder/handlebars')).catch(error => {
+  return KssBuilderBase.loadBuilder(options.builder ? KssBuilderBase.builderResolve(options.builder) : require('../builder/handlebars')).catch(error => {
     let catchLoadFailure;
     const supportedBuilders = [
       'builder/twig',

